README.md

```diff
@@ -182,13 +182,15 @@ A user can be allowed to resolve a task in three ways:
 
-| Name            | Description                                                                                           | Reference                                                |
-| --------------- | ----------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
-| **`Golang`**    | Builtin functions from Golang text template                                                           | [Doc](https://golang.org/pkg/text/template/#hdr-Actions) |
-| **`Sprig`**     | Extended set of functions from the Sprig project                                                      | [Doc](https://masterminds.github.io/sprig/)              |
-| **`field`**     | Equivalent to the dot notation, for entries with forbidden characters                                 | ``{{field `config` `foo.bar`}}``                         |
-| **`eval`**      | Evaluates the value of a template variable                                                            | ``{{eval `var1`}}``                                      |
-| **`evalCache`** | Evaluates the value of a template variable, and cache for future usage (to avoid further computation) | ``{{evalCache `var1`}}``                                 |
+| Name               | Description                                                                                                                | Reference                                                |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| **`Golang`**       | Builtin functions from Golang text template                                                                                | [Doc](https://golang.org/pkg/text/template/#hdr-Actions) |
+| **`Sprig`**        | Extended set of functions from the Sprig project                                                                           | [Doc](https://masterminds.github.io/sprig/)              |
+| **`field`**        | Equivalent to the dot notation, for entries with forbidden characters                                                      | ``{{field `config` `foo.bar`}}``                         |
+| **`eval`**         | Evaluates the value of a template variable                                                                                 | ``{{eval `var1`}}``                                      |
+| **`evalCache`**    | Evaluates the value of a template variable, and cache for future usage (to avoid further computation)                      | ``{{evalCache `var1`}}``                                 |
+| **`fromJson`**     | Decodes a JSON document into a structure. If the input cannot be decoded as JSON, the function will return an empty string | ``{{fromJson `{"a":"b"}`}}``                             |
```

Comment:
I think it'd be interesting to describe a common use case for this here

Reply:
Yes. I guess, the most common use case we could document is to return a JSON stringified data structure from a JavaScript expression (object, array), and use one of its members in the template.
Example: ``{{(eval `myExpression` | fromJson).myArr}}`` or ``{{(eval `myExpression` | fromJson).myObj}}``.